### ISSUE TITLE
Mermaid Gantt diagrams are fixed to 640px width

### ISSUE THREAD

**Issue by pyriand3r** [laurent22/joplin#6711]
When writing a bigger mermaid gantt diagram in Joplin i came across, that the diagram stopped growing at some point. Further research revealed, that it seems that the gantt diagram is somehow fixed to `max-width: 640px`.

I tried to add a `<style>` tag to the note with the diagram, overwriting the max-width to 100%. But that won't help because the image itself is rendered to 640px. Changing the max-width to 100% only sets the svg-tag to 100% width centering the digram but it keeps its 640px width.

## Environment

Joplin version: 2.8.8
Platform: MacOS
OS specifics: Monterey 12.5

## Steps to reproduce

1. Create a note with a bigger mermaid gantt diagram (with a wide time range)
2. See in the render window that the gantt diagram is only 640px wide and will not expand further
3. add following style code to the note to see the diagram center but not expand 
  ```
  <style>
	.mermaid svg {
		max-width: 100% !important;
	}
</style>
```

## Describe what you expected to happen

I would expect the diagram to use the width it needs to render the content correctly and readable, being able to be scrolled horizontally, or at least take as much space as possible.

### without css
<img width="2552" alt="mermaid_no_css" src="https://user-images.githubusercontent.com/2619815/182602395-0b4537ab-e4ed-4389-9a31-f1d149533d75.png">

### with css
<img width="2553" alt="mermaid_with_css" src="https://user-images.githubusercontent.com/2619815/182602458-24f0fcd1-4a79-43ce-abea-d822d4acc5a1.png">

**Comment by davelopware-davea** [comment]
If you check the computed styles you may find it is actually overriding the max-width - however by default the svg will maintain it's aspect ratio. When I modify the height in css too, I find the gantt chart fills the full width - try adding:

```
<style>
  .mermaid svg {
	height: auto;
	max-width:100% !important;
  }
</style>
```

**Comment by tessus** [comment]
I am not sure, if I can replicate this. Can you please provide a gantt chart.

**Comment by davelopware-davea** [comment]
try this simple example for the gantt chart:
```
\`\`\`mermaid
gantt
dateFormat  YYYY-MM-DD
title Adding GANTT diagram to mermaid
excludes weekdays 2014-01-10

section A section
Completed task        :done,    des1, 2014-01-06,2014-01-08
Active task           :active,  des2, 2014-01-09, 3d
Future task           :         des3, after des2, 5d
Future task2          :         des4, after des3, 5d
Future task3          :         des5, after des4, 5d
Future task4          :         des6, after des5, 5d
\`\`\`
```

(note I'm escaping the backticks so that they show in the github comment text - obvs replace \\\` with just a single \` )

now I get the following:

![image](https://user-images.githubusercontent.com/106663797/194342644-1fcbaa75-97ac-4f04-88f3-ba3646c6ca69.png)

Which is always limited in width in the rendered view. If I prepend the styling I mentioned on the 10 Aug:
```
<style>
  .mermaid svg {
	height: auto;
	max-width:100% !important;
  }
</style>
```
then you see a much more useful full-width gantt chart:

![image](https://user-images.githubusercontent.com/106663797/194343022-b32ff2c2-35ec-494a-b899-d71459be17bf.png)

**Comment by tessus** [comment]
Fixed by #8890
